### PR TITLE
Rating component

### PR DESCRIPTION
**Pull request by andreatosato**
Rating work only with Fontawesome because Star (full) and (empty) has same class name but difference is "regular" or "solid" icon style.

I want to color Icon but not work preperly

### CODE REVIEW

**Comment by andreatosato** [comment]
I add some feature and demo for Material Icon.

I need to develop:
1. Color icon
2. Disabled component

@stsrki  Can you help me in this 2 feature?
Can you review code and suggest me?
Thanks a lot

**Comment by stsrki** [comment]
Hi @andreatosato, I made some more changes so please test it.

I also plan to adjust colors to be applied from theme settings.

For Color and Size I still need to decide if it makes sense to add it now or postpone that feature for some other time.

**Comment by andreatosato** [comment]
Color is very hardcoded feature. But for now I think that coverage many use cases.
I fix material sample and test each platform (Bootstrap and Material).
Can you merge this feature?

**Comment by stsrki** [comment]
I will need to write documentation before I merge. And will try to add validation support. It might come in handy.

**Comment by andreatosato** [comment]
Can I help you?
Is there guideline?

**Comment by stsrki** [comment]
You mean with documentation? If you want you can write it under `docs/_docs/components` path. Just create a new md file. Look for other files as an example. Once you Submit I will review. Thanks

**Comment by andreatosato** [comment]
I add draft docs.

File: docs/_docs/components/rating.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+title: "Rating component"
+permalink: /docs/components/rating/
+excerpt: "Learn how to use rating component."
+toc: true
+toc_label: "Guide"
+---
+
+## Basic Memo
```
Comment (stsrki) [inline docs/_docs/components/rating.md:9]: There are still some descriptions for memo component. Can you fix it?

**Review by stsrki — CHANGES_REQUESTED** [review]

**Comment by stsrki** [comment]
I think the PR is now ready to be merged. @andreatosato can you test it once last time, please?

**Pull request merged**